CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 
 - #### Remove unused GraphQL resolvers locationById and locationsByParent
 
+- #### Remove unused GraphQL type `user.catchmentArea` in favor of `user.primaryOffice`
+
```

Comment:
do we actually wanna deprecate these or allow getting the catchmentArea via the hierarchy route?

Reply:
It's fine to do breaking changes between minors